corehq/apps/cloudcare/templates/cloudcare/cloudcare_home.html

```diff
@@ -185,6 +187,17 @@
         <section id="case-crumbs" style="width: 800px"></section>
         <section id="cases"></section>
         <section id="webforms" style="min-width: 800px;"></section>
+        {% if request|toggle_enabled:"INSTANCE_VIEWER" %}
+            <hr/>
+            <div id="instance-xml-home" style="min-width: 800px;" name="instance-xml-home">
```

Comment:
there's a `webforms.css` file that gets loaded on this page that you can put the styles in. may be worth also using some of the bootstrap classes instead of `min-width`